### PR TITLE
add profile page of myself

### PR DESCRIPTION
**Pull request by SpecialAir123**
### Description
It added my profile
 
### Issues Resolved
N/A

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by SpecialAir123** [comment]
I added myself as a new author

**Review by cwperks — CHANGES_REQUESTED** [review]
Hi @SpecialAir123 , I am seeing an empty file. Did you make sure to add any unstaged changes before committing?

You can check if you have unstaged changes using `git status`

If there is any change to check in then use `git add <filename>` followed by committing and pushing the change.

**Comment by SpecialAir123** [comment]
> Hi @SpecialAir123 , I am seeing an empty file. Did you make sure to add any unstaged changes before committing?
> 
> You can check if you have unstaged changes using `git status`
> 
> If there is any change to check in then use `git add <filename>` followed by committing and pushing the change.

I have committed a new version with my file

File: _authors/osci_2023/hongyuankang.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Hongyuan Kang
+short_name: Wendell
+twitter: N/A
+github: SpecialAir123
+---
+
+**<Hongyuan Kang>** <I am a senior CS student of UC Berkeley>
```
Comment (dtaivpp) [inline _authors/osci_2023/hongyuankang.markdown:8]: Can you remove the twitter line and remove the `<>` from your bio? Thank you!

**Review by dtaivpp — CHANGES_REQUESTED** [review]

File: _authors/osci_2023/hongyuankang.markdown [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: Hongyuan Kang
+short_name:  Hongyuan
```
Comment (joshuarrrr) [inline _authors/osci_2023/hongyuankang.markdown:3]: nit
```suggestion
short_name: Hongyuan
```

Comment (SpecialAir123) [reply _authors/osci_2023/hongyuankang.markdown]: I have modified it.

**Review by joshuarrrr — APPROVED** [review]
@SpecialAir123 All the commits need a signoff - follow the instructions in https://github.com/opensearch-project/project-website/pull/2157/checks?check_run_id=17157105376 to fix.

**Comment by SpecialAir123** [comment]
> @SpecialAir123 All the commits need a signoff - follow the instructions in https://github.com/opensearch-project/project-website/pull/2157/checks?check_run_id=17157105376 to fix.



> @SpecialAir123 All the commits need a signoff - follow the instructions in https://github.com/opensearch-project/project-website/pull/2157/checks?check_run_id=17157105376 to fix.

I have done that.

**Comment by SpecialAir123** [comment]
> @SpecialAir123 All the commits need a signoff - follow the instructions in https://github.com/opensearch-project/project-website/pull/2157/checks?check_run_id=17157105376 to fix.

I got this error when doing the rebase 
"could not apply 24711d43... add new osci author d-wwwang"

**Comment by joshuarrrr** [comment]
> > @SpecialAir123 All the commits need a signoff - follow the instructions in https://github.com/opensearch-project/project-website/pull/2157/checks?check_run_id=17157105376 to fix.
> 
> I got this error when doing the rebase "could not apply [24711d4](https://github.com/opensearch-project/project-website/commit/24711d43fb583fa5dc7f2c6243d848e86ac6d20e)... add new osci author d-wwwang"

OK, probably the easiest fix here is a soft reset which will squash all you commits into one (which you will sign):
```
git reset --soft HEAD~9 &&
git commit
```

**Comment by SpecialAir123** [comment]
> > > @SpecialAir123 All the commits need a signoff - follow the instructions in https://github.com/opensearch-project/project-website/pull/2157/checks?check_run_id=17157105376 to fix.
> > 
> > 
> > I got this error when doing the rebase "could not apply [24711d4](https://github.com/opensearch-project/project-website/commit/24711d43fb583fa5dc7f2c6243d848e86ac6d20e)... add new osci author d-wwwang"
> 
> OK, probably the easiest fix here is a soft reset which will squash all you commits into one (which you will sign):
> 
> ```
> git reset --soft HEAD~9 &&
> git commit
> ```
I dont know why I got so many commits

**Comment by SpecialAir123** [comment]
> > > @SpecialAir123 All the commits need a signoff - follow the instructions in https://github.com/opensearch-project/project-website/pull/2157/checks?check_run_id=17157105376 to fix.
> > 
> > 
> > I got this error when doing the rebase "could not apply [24711d4](https://github.com/opensearch-project/project-website/commit/24711d43fb583fa5dc7f2c6243d848e86ac6d20e)... add new osci author d-wwwang"
> 
> OK, probably the easiest fix here is a soft reset which will squash all you commits into one (which you will sign):
> 
> ```
> git reset --soft HEAD~9 &&
> git commit
> ```

I am so sorry but I really dont know what happened.

**Pull request closed without merging**